recipes/genehunter/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: ghm
+  version: "3.1"
+
+source:
+  fn: ghm-3.1.tgz
+  url: https://www.helmholtz-muenchen.de/fileadmin/GENEPI/downloads/ghm-3.1.tgz
+  md5: 0cfdc1aa80e7501f4766fa9537f96897
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - gcc   # [not osx]
+    - llvm  # [osx]
```

Comment:
libgcc as runtime dep is missing.
Please look at the docs ;) 
https://bioconda.github.io/guidelines.html#c-c